configuration/src/gas/defaults.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{
+    BridgeGasConfig, BridgeRouterGasLimits, ConnectionManagerGasLimits, CoreGasConfig,
+    EthHelperGasLimits, HomeGasLimits, HomeUpdateGasLimit, NomadGasConfig, ReplicaGasLimits,
+};
+use serde::{Deserialize, Deserializer};
+
+#[derive(Debug, Clone, Copy, serde::Serialize, PartialEq)]
+pub struct EvmDefaultWrapper(pub NomadGasConfig);
+
+impl<'de> Deserialize<'de> for EvmDefaultWrapper {
+    fn deserialize<D>(_: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        Ok(Self(EVM_DEFAULT))
+    }
+}
+
+pub const EVM_DEFAULT: NomadGasConfig = NomadGasConfig {
```

Comment:
should this be a default impl instead of a static?

Reply:
No longer using enum variant or wrapper so just leaving as a static